conf/config_ci_classification_local.yaml

```diff
@@ -25,6 +26,7 @@ sample:
   samples_dist: 200
   min_annotated_percent: 10 # Min % of non background pixels in stored samples. Default: 0
   mask_reference: False
+  scale_data: [0,1]
```

Comment:
`scale_data` should be a global parameter, since it has to be used for `images_to_samples.py` and `inference.py`.